src/components/htmlVideoPlayer/plugin.js

```diff
@@ -1722,6 +1724,37 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
         }];
     };
 
+    HtmlVideoPlayer.prototype.setPlaybackRate = function (val) {
+        var mediaElement = this._mediaElement;
+        if (mediaElement) {
+            mediaElement.playbackRate = val;
+        }
+        this._currentPlaybackRate = val;
+    };
+
+    HtmlVideoPlayer.prototype.getPlaybackRate = function () {
+        return this._currentPlaybackRate || 1.0;
+    };
+
+    HtmlVideoPlayer.prototype.getSupportedPlaybackRates = function () {
+        return [{
+            name: '1x',
+            id: 1.0
+        }, {
+            name: '1.25x',
+            id: 1.25
+        }, {
+            name: '1.5x',
+            id: 1.5
+        }, {
+            name: '1.75x',
+            id: 1.75
+        }, {
+            name: '2x',
+            id: 2.0
+        }];
```

Comment:
Could we also do half speed? it could be useful

Reply:
Sure, added 0.5x and 0.75x, since YouTube supports those too, which probably means they're useful.